Exercises/numbers/numbers.js

```diff
@@ -0,0 +1,53 @@
+//1. Assign by hand same value to 3 variables in binary, octal and hexadecimal format.
+//const decimal = new Number('99');
+
+const binary = new Number('1100011');
+const octal = new Number('143');
+const hexadecimal = new Number(63);
+
+console.log(binary,octal,hexadecimal);
+
+//2. Implement function `extractNumbers` which will leave only numbers in passed string (in the beginning of the string).
+
+function extractNumbers () {
+  price = '999usd';
+  number = parseInt(price,10); 
+  console.log(number);
+}
+extractNumbers();
+
+//3. Declare a variable with decimal value and convert to binary, octal and hexadecimal.
+
+const numberDecimal = new Number('9');
+
+console.log((numberDecimal).toString(2)); //1001
+console.log((numberDecimal).toString(8)); //011
+console.log((numberDecimal).toString(16)); //9
+
+//4. Use `Math.random` to generate random value from `0` to `100`.
+
+let randomInteger = Math.floor(Math.random() * 101);
+console.log(randomInteger);
+
+//5. Make function which will take a array with numbers as a parameter and calculate average number and round to hundredths.
+
+function arrayCalculation () {
+  values = [101,202,303,404,505];
```

Comment:
don't forget to use `let/const`. Never create variable without them, otherwise they will propagate to global scope and may conflict with other variables.

Reply:
got it. thanks, added let for variables in the function